### ISSUE TITLE
ValueError with pspnet.py

### ISSUE THREAD

**Issue by nhabili** [Vladkryvoruchko/PSPNet-Keras-tensorflow#17]
Hi, when I run pspnet.py, I get a ValueError due to a negative dimension (output file with error 
[pspnet.txt](https://github.com/Vladkryvoruchko/PSPNet-Keras-tensorflow/files/1309925/pspnet.txt)
attached). I'm using virtualenv with python2.7, tensorflow version 1.3.0 and keras version 2.0.8. I'm also using the weight files that you've provided.

Any help would be greatly appreciated!

Thanks
Nariman

**Comment by jmtatsch** [comment]
For me building the network still works.
can you print the input shapes in:
--> 177     prev_layer = AveragePooling2D(kernel, strides=strides)(prev_layer)

**Comment by nhabili** [comment]
Shapes are:

kernel = (60, 60)
strides = (60, 60)
prev_layer = (?, 2048, 60, 1)

**Comment by jmtatsch** [comment]
For me that is:

(60, 60)
(60, 60)
(?, 60, 60, 2048)

are you using theano dim ordering?

**Comment by Vladkryvoruchko** [comment]
For me everything also works. Have you done any modifications to the code, or

**Comment by nhabili** [comment]
The issue was that "image_data_format" in keras.json was set to "channels_first". I removed that line and I don't have that problem anymore. Thanks for your help.
